tests/unit/config/test_configfiles.py

```diff
@@ -223,6 +223,18 @@ def test_merge_persist(self, yaml, autoconfig, persist):
         mode = 'persist' if persist else 'normal'
         assert data['tabs.mode_on_change']['global'] == mode
 
+    @pytest.mark.parametrize('hide_decoration', [True, False])
+    def test_decoration_hiding(self, yaml, autoconfig, hide_decoration):
+        """Tests for migration of window.hide_wayland_decoration"""
```

Comment:
This should have a period at the end (but is probably not needed anymore anyways, see my comment above).